src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
+(e.g. `Repetition Time`, `Echo Time`, `Inversion Time` and `Flip Angle`). For 
+example, in a spoiled gradient-echo scan, keeping the repetition and the echo time 
+short with a relatively large flip angle increases the contribution of `T1` effects
+to the image contrast, yielding a primarily T1-weighted image.
+
+#### Grouped scan collections for contrast improvement or quantitative map calculation
+
+A group of anatomical scans may be collected to enhance certain contrast features,
+such as to calculate a weighted average of multi-echo gradient echo (`MEGRE`) images,
+which is known to improve segmentation algorithm outputs.
+
+Quantitative MRI (qMRI) methods mathematically characterize the signal changes observed
+in a collection of parametrically altered scans under certain biophysical model assumptions.
+These `grouped scan collections` can then be processed to derive a `qMRI map`, representing
+anatomical features in a physically meaningful parameter range.
+
+In both cases, the contrast characteristics change with varying acquisition parameters across
+`grouped scan collections`. Therefore, it is not tenable to name all the members
+of a `grouped scan collection` with one conventional MRI suffix label (e.g., `T1w`)
+or any other label that implies interchangeability among its instances. 
+
+To circumvent this problem, the `_<suffix>` entity is adaptively used according 
+to the intended application of anatomical imaging data. For conventional MRI 
+applications, the suffixes will correspond to common anatomical contrasts `T1w`,
+`T2w`, `T2starw` etc. For groups of scans acquired for contrast improvement or 
+qMRI processing, the `_<suffix>` indicates the collection that the scans belong to.
+
+#### The `_suffix` entity
+
+To ensure an comprehensible, human readable directory that contains anatomical 
+imaging data, the `_<suffix>` entity can be used in one of three ways:
+
+1. Conventional MRI suffixes
+2. Grouping suffixes
+3. Quantitative MRI (qMRI) map suffixes
+
+This distinction was added to the specification on acceptance of the BEP001 
+proposal in version `1.4.0`. However, as a result, some  suffixes that were introduced
+into BIDS at an earlier point time are inconsistent with this typology: for example 
+because they are linked to a readout-sequence rather than a contrast, or because 
+they don't clearly distinguish between a quantitative map or a contrast-weighted 
+image. These `legacy`-suffixes are no longer recommended but remain part of the 
+specification in order to maintain backwards compatability with previous versions 
+of the specification.
+
+These can be found in the [legacy suffixes](#legacy-suffixes) subsection.
+
+Previous versions of the specification used the term `modality_label` instead of 
+`_suffix` to represent this entity. The change in term was introduced in version 
+`1.4.0.` to accodomate a broad definition of anatomical imaging applications.
+
+##### Conventional MRI suffixes
+
+**Function:**
```

Comment:
Is function the best word here?  That is, within the qMRI community is "function" a unique and unambiguous term for the predominant contrast in an image?   If not, consider change to "Predominant contrast" or some other term that will not be confused with the dominant interpretation related to the term "functional."
```suggestion
**Purpose:**
```

Reply:
@Gilles86 - if you give this a :+1: vote then I think we can go ahead and commit

Reply:
Yes, but then also downstairs at qmri map suffixes